pages/en/lb4/Context.md

```diff
@@ -12,57 +12,73 @@ summary:
 
 - An abstraction of all state and dependencies in your application.
 - Context is what LB uses to "manage" everything.
-- A global registry for anything/everything in your app (all configs, state, dependencies, classes, etc).
+- A global registry for anything/everything in your app (all configs, state,
+dependencies, classes, etc).
 - An IoC container used to inject dependencies into your code.
 
 Why is it important to you?
 
-- You can use the context as a way to give loopback more "info" so that other dependencies in your app may retrieve it (ie. a centralized place/global builtin/in-memory storage mechanism).
-- LoopBack can help "manage" your resources automatically (through [Dependency Injection](Dependency-injection.html) and decorators).
-- You have full access to updated/real-time application+request state at all times.
+- You can use the context as a way to give loopback more "info" so that other
+dependencies in your app may retrieve it (ie. a centralized place/global
+builtin/in-memory storage mechanism).
+- LoopBack can help "manage" your resources automatically (through
+[Dependency Injection](Dependency-injection.html) and decorators).
+- You have full access to updated/real-time application+request state at all
+times.
 
 LoopBack supports two types of context: Application-level and Request-level
 
 ## Application-level context (global)
 
-- stores all the initial and modified app state throughout the entire life of the app (while the process is alive)
-- Generally configured when the application is created (though other things may modify things in the context while alive/running)
+- stores all the initial and modified app state throughout the entire life of
+the app (while the process is alive)
+- Generally configured when the application is created (though other things may
+modify things in the context while alive/running)
 
 Here is a simple example:
 
 ```js
 const Application = require('@loopback/core').Application;
 const app = new Application(); // `app` is a "Context"
-class MySequence { ... }
-app.sequence(MySequence);
+class MyController { ... }
```

Comment:
Can we revert this since either can be used as an example, but in the `Request-level context` section we expand on `MySequence`?